src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawerAdapterToJobs2dDriver.java

```diff
@@ -6,13 +6,14 @@
 import edu.kis.powp.jobs2d.Job2dDriver;
 
 /**
- * driver adapter to drawer with several bugs.
+ * Driver adapter to drawer.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class DrawerAdapterToJobs2dDriver implements Job2dDriver {
```

Comment:
Not the other way around?  Jobs2dDriverToDrawerAdapter? 
BTW: The name does not need to mention the adapter pattern (it is not the design issue but only solution to the problem).

Reply:
Ok, now should be fixed.